setup.py

```diff
@@ -26,6 +26,7 @@
     'recommonmark>=0.4.0',
     'Sphinx>=1.3.5',
     'sphinxcontrib-napoleon>=0.4.4',
+    'sphinx-rtd-theme>=0.1.9',
```

Comment:
@sbellem I just double checked with @ttmc, ReadTheDocs expects a `requirements.txt` to install dependencies. I'm writing them to ask if they support `extras_require` in `setup.py`, so we can keep everything in the same place.